Upgrade Kiali chart (#14589)
kyma-project/kyma: resources/kiali/templates/NOTES.txt

> Welcome to Kiali! For more details on Kiali, see: https://kiali.io

Reviewer: ```suggestion
Welcome to Kiali! For more details, see: https://kiali.io
```

> The Kiali Server [{{ .Chart.AppVersion }}] has been installed in namespace [{{ .Release.Namespace }}]. It will be ready soon.

Reviewer: ```suggestion
The Kiali Server [{{ .Chart.AppVersion }}] has been installed in Namespace [{{ .Release.Namespace }}]. Wait a minute for it to start.
```
